scripts/integtest.sh

```diff
@@ -98,9 +98,12 @@ then
   leader=$(echo  $data | cut -d ',' -f1 | cut -d ':' -f1,2 )
   follower=$(echo $data  |  cut -d ',' -f2 | cut -d ':' -f1,2 )
```

Comment:
Could we log the values and attach the output as well? (for future debugging and review)

Reply:
hosts and port (along with transport)